scripts/deploy.ps1

```diff
@@ -0,0 +1,5 @@
+python setup.py sdist bdist_wheel
```

Comment:
Same question, what does this file do ?

Reply:
This file is used to deploy the package to Pypi. It more or less follows [this](https://realpython.com/pypi-publish-python-package/). See the author suggests there `bumpversion` package to ease the release cycle.

Reply:
Like I said, no intention to push on Pypi currently.

Reply:
This file was removed in the latest pull request draft.